pkg/util/fsnotification/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package fsnotification provides helper functions that wrapper the fsnotify filesystem
```

Comment:
wrapper as a verb.


On Fri, Apr 1, 2016 at 8:55 PM, Rodolfo Carvalho <notifications@github.com>
wrote:

> In pkg/util/fsnotification/doc.go
> <https://github.com/openshift/origin/pull/8268#discussion_r58285166>:
>
> > @@ -0,0 +1,3 @@
> > +// Package fsnotification provides helper functions that wrapper the fsnotify filesystem
>
> it appears to be a bit of a slang usage.
> Which one usage is slang?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8268/files/c2a67b7df978af31a95aa3078b18b62d496565ae..1b1430c569d988b63b595305242992d18f292285#r58285166>
>



-- 
Ben Parees | OpenShift